c_binding/include/libstoragemgmt/libstoragemgmt_plug_interface.h

```diff
@@ -1030,6 +1031,138 @@ struct lsm_ops_v1_2 {
     lsm_plug_volume_raid_create vol_create_raid;
 };
 
+/**
+ * New in version 1.3.
+ * Retrieve a list of batteries.
+ * @param[in]   c               Valid lsm plug-in pointer
+ * @param[in]   search_key      Search key
+ * @param[in]   search_value    Search value
+ * @param[out]  bs              Array of batteries
+ * @param[out]  count           Number of batteries
+ * @param[in]   flags           Reserved
+ * @return LSM_ERR_OK, else error reason
+ */
+typedef int (*lsm_plug_battery_list) (lsm_plugin_ptr c,
+                                      const char *search_key,
+                                      const char *search_val,
+                                      lsm_battery ***bs,
+                                      uint32_t *count, lsm_flag flags);
+
+/**
+ * New in version 1.3.
+ * Provides for batteries filtering when an storage system doesn't support this
```

Comment:
Maybe something more like "Allows for battery filtering when a storage system can't support it natively."